tests/node/app-boot-test.js

```diff
@@ -112,53 +144,49 @@ QUnit.test("It is possible to render a view with a nested {{view}} helper in Nod
     })
   });
 
-  var morph = {
-    contextualElement: {},
-    setContent: function(element) {
-      this.element = element;
-    }
-  };
-
   var view = View.create({
-    _domHelper: new DOMHelper(new SimpleDOM.Document()),
-    _morph: morph
+    _domHelper: new DOMHelper(new SimpleDOM.Document())
   });
 
-  var renderer = view.renderer;
-
-  Ember.run(function() {
-    renderer.renderTree(view);
-  });
+  run(view, view.createElement);
 
   var serializer = new SimpleDOM.HTMLSerializer(SimpleDOM.voidMap);
-  ok(serializer.serialize(morph.element).match(/<h1>Hello World<\/h1> <div><div id="(.*)" class="ember-view"><p>The files are \*inside\* the computer\?\!<\/p><\/div><\/div>/));
+  ok(serializer.serialize(view.element).match(/<h1>Hello World<\/h1> <div><div id="(.*)" class="ember-view"><p>The files are \*inside\* the computer\?\!<\/p><\/div><\/div>/));
 });
 
-function createApplication() {
-  var App = Ember.Application.extend().create({
-    autoboot: false
-  });
+QUnit.test("It is possible to render a view with {{link-to}} in Node", function() {
+  QUnit.stop();
 
-  App.Router = Ember.Router.extend({
-    location: 'none'
+  var app;
+
+  run(function() {
+    app = createApplication();
+
+    app.Router.map(function() {
+      this.route('photos');
+    });
+
+    registerDOMHelper(app);
+    registerTemplates(app, {
+      application: "<h1>{{#link-to 'photos'}}Go to photos{{/link-to}}</h1>"
+    });
   });
 
-  return App;
-}
+  app.visit('/').then(function(instance) {
+    QUnit.start();
 
-QUnit.test("It is possible to render a view with {{link-to}} in Node", function() {
+    var element = renderToElement(instance);
+
+    assertHTMLMatches(element.firstChild, /^<div id="ember\d+" class="ember-view"><h1><a id="ember\d+" class="ember-view" href="\/photos">Go to photos<\/a><\/h1><\/div>$/);
+  });
+});
+
+QUnit.test("It is possible to render outlets in Node", function() {
+  QUnit.stop();
```

Comment:
There are two calls to `visit`, and I presume they don't know which will finish last. `stop` twice just increments a number twice, `start` decrements it back to zero and the test can complete.